pyproject.toml

```diff
@@ -274,6 +281,10 @@ omit = [
     "tests/",
     "src/telegram/__main__.py"
 ]
+# Relevant for python 3.14: https://github.com/nedbat/coveragepy/issues/1983
```

Comment:
can you please clarify what should be done with this addition once the linked issue is resolved? e.g. should we remove that setting?

Reply:
that issue is resolved and now we can remove that